CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+
+## Filing a issue
+### Before filing an issue
+
+If you are unsure whether you have found a bug, please consider asking in the [Slack](https://kubernetes.slack.com/messages/openebs) first. If
+the behavior you are seeing is confirmed as a bug or issue, it can easily be re-raised in the [issue tracker](https://github.com/openebs/velero-plugin/issues).
+
+### Filing issues
+
+When filing an issue, make sure to answer these seven questions:
+
+1. What version of Velero are you using (`velero version`)?
+2. What version of OpenEBS are you using?
+3. What version of velero-plugin are you using?
+4. What steps you followed to create a backup or restore?
+5. What did you expect to see?
+6. What did you see instead?
+7. Logs of velero/velero pod and openebs/maya-apiserver pod.
+
+## Steps to Contribute
+velero-plugin is an Apache 2.0 Licensed project and all your commits should be signed with Developer Certificate of Origin. See [Sign your work](#sign-your-work).
+
+For setting up a Development environment on your local host, see the detailed instructions [here](developer-setup.md).
+
+* Find an issue to work on or create a new issue. The issues are maintained at [openebs/velero-plugin](https://github.com/openebs/velero-plugin/issues). You can pick up from a list of [good-first-issues](https://github.com/openebs/velero-plugin/labels/good%20first%20issue).
+* Claim your issue by commenting your intent to work on it to avoid duplication of efforts.
+* Fork the repository on GitHub.
+* Create a branch from where you want to base your work (usually master).
+* Commit your changes by making sure the commit messages convey the need and notes about the commit.
+* Please make sure than your code is aligned with the standard mentioned at [code-standard](code-standard.md).
+* Verify that your changes passes `make lint` or `make lint-docker` (docker version of `make lint`)
+* Push your changes to the branch in your fork of the repository.
+* Submit a pull request to the original repository. See [Pull Request checklist](#pull-request-checklist)
+
+## Pull Request Checklist
+* Rebase to the current master branch before submitting your pull request.
+* Commits should be as small as possible. Each commit should follow the checklist below:
+  - For code changes, add tests relevant to the fixed bug or new feature.
+  - Commit header (first line) should convey what changed
+  - Commit body should include details such as why the changes are required and how the proposed changes help
+  - DCO Signed, please refer [signing commit](code-standard.md/sign-your-commits)
+* If your PR is about fixing a issue or new feature, make sure you add a change-log. Refer [Adding a Change log](code-standard.md/adding-a-changelog)
+* PR title must follow convention: `<type>(<scope>): <subject>`.
+
+  For example:
+  ```
+   feat(backup) : support for backup to aws
```

Comment:
we need to remove 'space' between ')' and ':'
feat(backup): support for backup to aws